src/tribler-common/tribler_common/tests/test_dependencies.py

```diff
@@ -37,7 +37,8 @@ async def test_get_dependencies():
     # assert that in each scope dependencies are exist
     assert list(get_dependencies(Scope.gui))
     assert list(get_dependencies(Scope.core))
-    assert list(get_dependencies(Scope.common))
+    # Disabled because the list of common dependencies is currently empty
+    #assert list(get_dependencies(Scope.common))
```

Comment:
**Nit** you can safely remove these lines.

Reply:
Done